yogstation/code/modules/antagonists/gang/gang.dm

```diff
@@ -0,0 +1,475 @@
+/datum/antagonist/gang
+	name = "Gangster"
+	roundend_category = "gangsters"
+	can_coexist_with_others = FALSE
+	job_rank = ROLE_GANG
+	antagpanel_category = "Gang"
+	var/hud_type = "gangster"
+	var/message_name = "Gangster"
+	var/datum/team/gang/gang
+
+/datum/antagonist/gang/can_be_owned(datum/mind/new_owner)
+	. = ..()
+	if(.)
+		if(new_owner.unconvertable)
+			return FALSE
+
+/datum/antagonist/gang/apply_innate_effects(mob/living/mob_override)
+	var/mob/living/M = mob_override || owner.current
+	update_gang_icons_added(M)
+
+/datum/antagonist/gang/remove_innate_effects(mob/living/mob_override)
+	var/mob/living/M = mob_override || owner.current
+	update_gang_icons_removed(M)
+
+/datum/antagonist/gang/get_team()
+	return gang
+
+/datum/antagonist/gang/greet()
+	gang.greet_gangster(owner)
+
+/datum/antagonist/gang/farewell()
+	if(ishuman(owner.current))
+		owner.current.visible_message("<span class='deconversion_message'>[owner.current] looks like [owner.current.p_theyve()] just remembered [owner.current.p_their()] real allegiance!</span>", null, null, null, owner.current)
+		to_chat(owner, "<span class='userdanger'>You are no longer a gangster!</span>")
+
+/datum/antagonist/gang/on_gain()
+	if(!gang)
+		create_team()
+	..()
+	var/mob/living/carbon/human/H = owner.current
+	if(istype(H))
+		if(owner.assigned_role == "Clown")
```

Comment:
Please move the clown removal to the gangboss datum, that way regular gangsters are still clowns.